woocommerce/woocommerce-ios
# Order Details: navigate to Refunded Products detail screen

Closes #1700.

This PR displays the "Refunded Products" section on Order Details and a cell that counts the number of items that have been refunded. Tapping on the "N Items" cell navigates the user to the Refunded Product Details screen.

⚠️ Change base to `develop` before merging this PR.

### Screenshots

| Order Details | Refunded Products Detail | 
| --- | --- |
| <img src="https://user-images.githubusercontent.com/1062444/71753793-48f92800-2e49-11ea-80a4-c62da4c7adb0.png" width="350" /> | <img src="https://user-images.githubusercontent.com/1062444/71753827-66c68d00-2e49-11ea-978e-167fe03cc2d3.png" width="350" /> |

### To Test

1. On the website, choose an order and create a refund by item. Refund 2 items or more and add a note.
After creating the refund, go to the app and navigate to the Order Details for the chosen order.
2. The "Refunded Products" section title should appear and a "2 Items" detail cell should be in this section.
3. Tap the "2 Items" cell and navigate to the Refunded Products detail screen. The title should appear. The back button should not have any words.

Update release notes:
